backend/src/v5/processors/teamspaces/projects/models/commons/tickets.groups.js

```diff
@@ -79,22 +105,126 @@ const getObjectArrayFromRules = async (teamspace, project, model, revId, rules)
 	return { container: model, _ids: Object.values(matchedMeshes) };
 };
 
-TicketGroups.updateTicketGroup = updateGroup;
+const convert3dRepoIdsToExternalIds = async (teamspace, project, objects) => {
+	const convertedObjects = await Promise.all(objects.map(async (obj) => {
+		if (!obj._ids) {
+			return obj;
+		}
+
+		const convertedObject = { ...obj };
+
+		const shared_ids = await getNodesByIds(teamspace, project, obj.container, obj._ids,
```

Comment:
camel case (this is why we should not be turning off a linter rule on the whole file!)

Reply:
even without the linter rule disabling shared_ids is fine, it only complains when the _ is in the beggining, but I can change it to sharedIds regardless

Reply:
as per the comment below, change it to nodes or something.

sharedIds is not a good representative for what it is holding